### ISSUE TITLE
ndarray is not JSON serializable when exporting bw2package

### ISSUE THREAD

**Issue by mkvdhulst** [polca/premise#101]
Hi, 

I have applied premise 1.5.0-alpha8 on ecoinvent 3.9.1 and it worked like a charm. I created a superstructure database for several years and all three pathways of IMAGE. I can use the superstructure database and scenarios difference file in activity browser without any issues. However, when I try to export the database from activity browser as bw2package, it fails with the following error.

> Traceback (most recent call last):
  File "C:\Users\hulstmkvd\Anaconda3\envs\ab\lib\site-packages\activity_browser\ui\wizards\db_export_wizard.py", line 38, in accept
    self.perform_export()
  File "C:\Users\hulstmkvd\Anaconda3\envs\ab\lib\site-packages\activity_browser\ui\wizards\db_export_wizard.py", line 52, in perform_export
    EXPORTERS[export_as](db_name, out_path)
  File "C:\Users\hulstmkvd\Anaconda3\envs\ab\lib\site-packages\activity_browser\bwutils\exporters.py", line 135, in store_database_as_package
    ABPackage.unrestricted_export(db, out_file)
  File "C:\Users\hulstmkvd\Anaconda3\envs\ab\lib\site-packages\activity_browser\bwutils\importers.py", line 122, in unrestricted_export
    cls._write_file(path, [cls._prepare_obj(obj, False)])
  File "C:\Users\hulstmkvd\Anaconda3\envs\ab\lib\site-packages\bw2io\package.py", line 124, in _write_file
    JsonWrapper.dump_bz2(JsonSanitizer.sanitize(data), filepath)
  File "C:\Users\hulstmkvd\Anaconda3\envs\ab\lib\site-packages\bw2data\serialization.py", line 43, in dump_bz2
    (JsonWrapper.dumps(data)).encode('utf-8')
  File "C:\Users\hulstmkvd\Anaconda3\envs\ab\lib\site-packages\bw2data\serialization.py", line 64, in dumps
    return json.dumps(data)
  File "C:\Users\hulstmkvd\Anaconda3\envs\ab\lib\json\__init__.py", line 231, in dumps
    return _default_encoder.encode(obj)
  File "C:\Users\hulstmkvd\Anaconda3\envs\ab\lib\json\encoder.py", line 199, in encode
    chunks = self.iterencode(o, _one_shot=True)
  File "C:\Users\hulstmkvd\Anaconda3\envs\ab\lib\json\encoder.py", line 257, in iterencode
    return _iterencode(o, 0)
  File "C:\Users\hulstmkvd\Anaconda3\envs\ab\lib\json\encoder.py", line 179, in default
    raise TypeError(f'Object of type {o.__class__.__name__} '
TypeError: Object of type ndarray is not JSON serializable

What I figured out from some googling is that it could probably be resolved by turning the array into a list with the .tolist() method from NumPy, but I am not sure where it should be added in the code. The last version of premise I used before 1.5.0-alpha 8 was 1.4.1 and I was, and still am, able to export superstructure databases created with that version to a bw2package. Therefore, I assume the error arises somewhere in the new code for premise 1.5.0.

**Comment by romainsacchi** [comment]
Hi @mkvdhulst ,

Sorry about that. Upgrading to `1.5.0-beta`and regenerating a database should solve the issue.

**Comment by mkvdhulst** [comment]
Hi @romainsacchi,

I have tried the latest build, `1.5.0-beta2`, but unfortunately I still get the same error. I noticed I did not have the latest version of `activity browser`, but updating also did not resolve the issue. The issue for me is really localized to superstructure databases created with recent versions of `premise`. I can export e.g. a regular ecoinvent database as _.bw2package_ with `activity browser` just fine and I can even export older premise-generated superstructure databases of ecoinvent to _.bw2package_, as long as I have generated them with an `premise 1.4.1` or older. The issue is not localized to my device/environment, since a colleague of mine tried as well and got the same error. 

_FYI, when using `premise 1.5.0-beta2` I also ran into an issue with generating scenarios for clinker, but I managed to resolve that and then it works just fine, besides the exporting issue. I will try to create a pull request for my solution of the clinker scenario issue as soon as I figured out how to do that._
